### PR TITLE
new: usr: Add dict schedule object arg to schedule pipeline based on the environment [FLYCS-814]

### PR DESCRIPTION
**Pull request by yannis-arg**

### CODE REVIEW

File: tests/test_pipeline.py [hunk 1]
```diff
@@ -216,6 +273,16 @@ def test_serialize_deserialize_pubsub(self, my_pipeline_pubsub, my_entity):
             if isinstance(my_pipeline_pubsub, ParametrizedPipeline):
                 assert loaded.params  # ensure the params area loaded
 
+    def test_serialize_deserialize_env(self, my_pipeline_env, my_entity):
```
Comment (christophedcpm) [inline tests/test_pipeline.py:276]: I don't really see the point of this new test.
Aren't you supposed to also test that the new schedule format does serialize and serialize properly ?

Comment (christophedcpm) [inline tests/test_pipeline.py:276]: I don't really see the point of this new test.
Aren't you supposed to also test that the new schedule format does serialize and serialize properly ?

**Review by christophedcpm — CHANGES_REQUESTED** [review]
LGTM, just one comment regarding a test that I do understand.

File: tests/test_pipeline.py (hunk 1 above)
Comment (yannis-arg) [reply tests/test_pipeline.py:276]: This test just show that the Pipeline object (with env schedule defined) can correctly being created & can be deserialized without breaking everything. But actually the **test_init_env** & **test_to_dict_env** is already testing this. Should I remove it ? @christophedcpm

**Review by christophedcpm — APPROVED** [review]

**Pull request merged**